bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/discovery/YeelightDiscoveryService.java

```diff
@@ -88,6 +88,8 @@ private ThingUID getThingUID(DeviceBase device) {
             case ceiling3:
                 return new ThingUID(YeelightBindingConstants.THING_TYPE_CEILING, device.getDeviceId());
             case ceiling1:
+            case ceil26:
```

Comment:
Is there a reason to not name it celling26 ?

Reply:
This is how device identify itself during network discovery. I have device response body in the main topic above. See "model" field.

Reply:
Ok